RxMarbles/OperatorsTableViewController.swift

```diff
@@ -10,58 +10,26 @@ import UIKit
 import RxSwift
 import RxCocoa
 
-struct Section {
-    var name: String
-    var rows: [Operator]
-}
+class OperatorsTableViewController: UITableViewController {
+
+	private let viewModel = OperatorsTableViewModel()
 
-class OperatorsTableViewController: UITableViewController, UISearchResultsUpdating {
-    private let _disposeBag = DisposeBag()
+	private let _disposeBag = DisposeBag()
 
     var selectedOperator: Operator = Operator.combineLatest {
         didSet {
             tableView.reloadData()
         }
     }
-    
-    private let _searchController = UISearchController(searchResultsController: nil)
-    private var _filteredSections = [Section]()
-    
-    private let _sections = [
-        Section(
-            name: "Combining",
-            rows: [.combineLatest, .concat, .merge, .startWith, .switchLatest, .withLatestFrom, .zip]
-        ),
-        Section(
-            name: "Conditional",
-            rows: [.amb, .skipUntil, .skipWhile, .skipWhileWithIndex, .takeUntil, .takeWhile, .takeWhileWithIndex]
-        ),
-        Section(
-            name: "Creating",
-            rows: [.empty, .interval, .just, .never, .of, .repeatElement, .throw, .timer]
-        ),
-        Section(
-            name: "Error",
-            rows: [.catchError, .catchErrorJustReturn, .retry]
-        ),
-        Section(
-            name: "Filtering",
-            rows: [.debounce, .distinctUntilChanged, .elementAt, .filter, .ignoreElements, .sample, .single, .skip, .skipDuration, .take, .takeDuration, .takeLast, .throttle]
-        ),
-        Section(
-            name: "Mathematical",
-            rows: [.reduce]
-        ),
-        Section(
-            name: "Transforming",
-            rows: [.buffer, .delaySubscription, .flatMap, .flatMapFirst, .flatMapLatest, .map, .mapWithIndex, .scan, .toArray]
-        ),
-        Section(
-            name: "Utility",
-            rows: [.timeout]
-        )
-    ]
-    
+
+	private lazy var _searchController: UISearchController = {
```

Comment:
```suggestion
	private lazy var searchController: UISearchController = {
```

Seems unneeded, we have ACL so no need for underscore naming